src/index.html

```diff
@@ -1,12 +1,371 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="home-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" >
+        <img src="./images/logo.png" alt="logo"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="video nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+
+    </header>
+
+    <main class="cards home-page__cards">
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg" alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="container">
+          <div class="container__stars stars">
```

Comment:
```suggestion
          <div class="container__stars stars stars--4">
```
stars should have modifier which will be responsible for amount of active cards (as in prev tasks)